bin/make_deploy

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -euo pipefail
+
+gh workflow run --ref main deploy.yml > /dev/null
+GITHUB_TOKEN=$(gh auth token) watch_gha_runs --wait-for-start --only deploy 'https://github.com/hypothesis/test-pyramid-app' main
```

Comment:
`GITHUB_TOKEN` is needed to get around GitHub API rate limits that `watch_gha_runs` otherwise runs into very quickly. This uses GitHub CLI to set `GITHUB_TOKEN`